tetrateci/create_istio_release.sh

```diff
@@ -101,18 +102,13 @@ if [[ -z $TEST ]]; then
 
     go run main.go build --manifest manifest.archive.yaml
 
+    python3 -m pip install --upgrade cloudsmith-cli --user
+
     PACKAGES=$(ls /tmp/istio-release/out/ | grep "istio")
     for package in $PACKAGES; do
         echo "Publishing $package"
-        rm -f /tmp/curl.out
-        curl -T /tmp/istio-release/out/$package -u$BINTRAY_USER:$API_KEY $BINTRAY_API/$TAG/$package -o /tmp/curl.out
-        cat /tmp/curl.out
-        grep "success" /tmp/curl.out
+        python3 -m cloudsmith push raw ${CLOUDSMITH_USER}/getistio $package
```

Comment:
cloudsmith is the cli. Command has to be cloudsmith push raw ${CLOUDSMITH_USER}/getistio $package

Reply:
For the above to work, export CLOUDSMITH_API_KEY=whatever-from-secret

Reply:
That is already taken care of in the workflow.